### PR TITLE
Added tests!

### PR DESCRIPTION
**Pull request by czardoz**
Should merge #1 before this.

### CODE REVIEW

File: lib/runner/extensions/event.command.js [hunk 1]
```diff
@@ -161,7 +162,7 @@ module.exports = {
             }, function (err, results) {
                 // trigger the event completion callback
                 util.safeCall(this.triggers[eventName], this.triggers, null, cursor, results, item);
-                next(null, results, err);
```
Comment (czardoz) [inline lib/runner/extensions/event.command.js]: This corrects the existing behavior which treats a request error as fatal.

File: lib/runner/extensions/event.command.js [hunk 2]
```diff
@@ -93,6 +93,7 @@ module.exports = {
                     }),
                 stopOnScriptError = _.has(payload, 'stopOnScriptError') ? payload.stopOnScriptError :
                     this.options.stopOnScriptError,
+                abortOnError = this.options.abortOnError,
```
Comment (shamasis) [inline lib/runner/extensions/event.command.js]: Why also not take this from payload like stopOnError does?

File: lib/runner/extensions/event.command.js [hunk 3]
```diff
@@ -151,7 +152,7 @@ module.exports = {
                     this.triggers.script(err || null, cursor, result, script, event, item);
 
                     // move to next script and pass on the results for accumulation
-                    next((stopOnScriptError && err) ? err : null, _.assign({
+                    next(((stopOnScriptError || this.options.abortOnError) && err) ? err : null, _.assign({
```
Comment (shamasis) [inline lib/runner/extensions/event.command.js]: Why use `this.options.abortOnError` when `abortOnError` is defined on top?

File: lib/runner/extensions/event.command.js (hunk 1 above)
Comment (shamasis) [inline lib/runner/extensions/event.command.js]: Rewrite with `(abortOnError && err) ? err : null`

**Pull request merged**